<launch>

<!-- <arg name="behaviorTreeFile" default="$(find tutorials_btros)/trees/test_tree_1.xml"/> -->
<arg name="behaviorTreeFile" default="$(find tutorials_btros)/trees/mission_planner.xml"/>
<param name="behaviorTreeFile" value="$(arg behaviorTreeFile)"/>
<arg name="useGroot" default="1"/>

<node pkg="tutorials_btros" type="ros_action_server" name="ros_action_server" launch-prefix="xterm -e" required="true"/>
<node pkg="tutorials_btros" type="ros_service_server" name="ros_service_server" launch-prefix="xterm -e"  required="true"/>
<node pkg="tutorials_btros" type="mission_planner_bt" name="mission_planner_bt" output="screen"  required="true"/>
<!-- <node pkg="tutorials_btros" type="bt_executor" name="bt_executor" output="screen"  required="true"/> -->


<!-- GROOT (OPTIONAL) -->
<node  if="$(arg useGroot)" pkg="groot" type="Groot" name="Groot"  launch-prefix="xterm -e"  />

</launch>
